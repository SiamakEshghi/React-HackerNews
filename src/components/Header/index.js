import React from 'react';
import 'components/style.css';
import logo from 'img/logo.png'

export default () => {
    return (
        <div className="header">
            <img className="logo" src={logo}/>
            <h1>TOP NEWS WEBSITE</h1>
            <h5 className="desc">React App</h5>
        </div>
    );
}