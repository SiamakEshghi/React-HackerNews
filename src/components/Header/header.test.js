import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from 'components/Header';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<Header />);
});

it('Header snapshot test', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
})