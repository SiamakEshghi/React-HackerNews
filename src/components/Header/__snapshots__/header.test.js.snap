// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot test 1`] = `
<div
  className="header"
>
  <img
    className="logo"
    src="logo.png"
  />
  <h1>
    TOP NEWS WEBSITE
  </h1>
  <h5
    className="desc"
  >
    React App
  </h5>
</div>
`;
